roaster-testsuite/roaster-testsuite-jaxrs/src/test/java/hu/icellmobilsoft/roaster/testsuite/jaxb/JaxbGetOctetStreamIT.java

```diff
@@ -0,0 +1,101 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 - 2022 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.roaster.testsuite.jaxb;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mockserver.client.MockServerClient;
+import org.mockserver.model.HttpRequest;
+import org.mockserver.model.HttpResponse;
+import org.mockserver.model.MediaType;
+import org.testcontainers.containers.MockServerContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import hu.icellmobilsoft.roaster.jaxrs.response.producer.RestProcessor;
+import hu.icellmobilsoft.roaster.jaxrs.response.producer.impl.ConfigurableResponseProcessor;
+import hu.icellmobilsoft.roaster.weldunit.BaseWeldUnitType;
+
+/**
+ * Starts a mockserver with testcontainers to validate the jaxb rest calls
+ * 
+ * @author imre.scheffer
+ * @since 0.8.0
+ */
+@Tag("integration")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@DisplayName("Test of jaxb client GET octet stream")
+class JaxbGetOctetStreamIT extends BaseWeldUnitType {
+
+    @Inject
+    @RestProcessor(configKey = "testsuite.rest.test")
+    private ConfigurableResponseProcessor<String> processor;
+
+    @Inject
+    @RestProcessor(configKey = "testsuite.rest.test", expectedStatusCode = 500)
+    private ConfigurableResponseProcessor<String> processor500;
+
+    private static final MockServerContainer MOCK_SERVER = new MockServerContainer(DockerImageName.parse("mockserver/mockserver:mockserver-5.13.2"));
```

Comment:
A `@Container` annotációval elvileg automatán elindítja, és leállítja. TM4J esetén ezt nem lehetet használni, a teszt életciklusa miatt.

Reply:
Ezt probaltam de nem jartam sikerrel, nem inditotta a containert

Reply:
Szerintem @BeforeEach használatához csinálták az automata részt. De akkor maradhat így is.